python/ray/serve/_private/deploy_utils.py

```diff
@@ -70,14 +69,13 @@ def get_deploy_args(
     deployment_config.version = version
 
     controller_deploy_args = {
-        "name": name,
+        "deployment_name": name,
         "deployment_config_proto_bytes": deployment_config.to_proto_bytes(),
         "replica_config_proto_bytes": replica_config.to_proto_bytes(),
         "route_prefix": route_prefix,
         "deployer_job_id": ray.get_runtime_context().get_job_id(),
         "is_driver_deployment": is_driver_deployment,
         "docs_path": docs_path,
-        "app_name": app_name,
```

Comment:
why is this change relevant?

Reply:
I use `deployment_name` in most places so it's not confusing now that there's both application and deployment name, it's just a preference, I made the corresponding changes in other places so this doesn't break things.

I removed `app_name` because it wasn't actually used.